src/attributes/components/AttributeDetails/AttributeDetails.tsx

```diff
@@ -2,18 +2,18 @@ import { NumericUnits } from "@dashboard/attributes/components/AttributeDetails/
 import CardTitle from "@dashboard/components/CardTitle";
 import ControlledCheckbox from "@dashboard/components/ControlledCheckbox";
 import FormSpacer from "@dashboard/components/FormSpacer";
-import SingleSelectField from "@dashboard/components/SingleSelectField";
```

Comment:
Question: Is `SingleSelectField` no longer used at all in Dashboard? If so we can remove it from codebase.

Reply:
That's the plan, but I'd like to this In separate PR